scripts/templates/create-component/EmptyComponent.mustache

```diff
@@ -1,16 +1,13 @@
 import { {{componentName}}View } from './{{componentName}}.view';
-import { {{componentName}}Styles } from './{{componentName}}.styles';
+import { {{componentName}}Styles, {{componentName}}Tokens } from './{{componentName}}.styles';
 import { {{componentName}}State } from './{{componentName}}.state';
-import { I{{componentName}}Props, I{{componentName}}ViewProps, I{{componentName}}Styles } from './{{componentName}}.types';
+import { I{{componentName}}Props } from './{{componentName}}.types';
 import { createComponent } from '../../Foundation';
 
-export const {{componentName}}: React.StatelessComponent< I{{componentName}}Props> = createComponent<
-  I{{componentName}}Props,
-  I{{componentName}}ViewProps,
-  I{{componentName}}Styles
->({
+export const {{componentName}}: React.StatelessComponent< I{{componentName}}Props> = createComponent({
```

Comment:
nit: remove the extra spaces after open < in this and various other files?
```suggestion
export const {{componentName}}: React.StatelessComponent<I{{componentName}}Props> = createComponent({
```

Reply:
They were there because it shows as an error in the mustache parser in VSCode, but removing them doesn't seem to affect the output. Hmm